### DISCUSSION TITLE
How do i disable all options?

### DISCUSSION THREAD

**Discussion by vsjoqvist** [starship/starship#4097] [Q&A] [answered]
I basically only want 5 of the modules, and since most of the modules are on by default its super tedious to disable them all by hand, is there any setting like ``disable_all``?

**Comment by chipbuster** [comment, marked as answer]
If you write a top-level `format` string, you should be able to include only the modules that you want. Would this work for you?

**Reply by vsjoqvist** [reply]
I think so
